docs/documentation/server_admin/topics/clients/client-policies.adoc

```diff
@@ -29,15 +29,15 @@ Validation of client configurations::
     the advanced security requirements. In the future, individual client configuration settings may be replaced by Client Policies directly performing required validations.
 
 Conformance to a required security standards and profiles such as FAPI and OAuth 2.1::
-    The _Global client profiles_ are client profiles pre-configured in {project_name} by default. They are pre-configured to be compliant with standard security profiles like link:{adapterguide_link}#_fapi-support[FAPI] and link:{adapterguide_link}#_oauth21-support[OAuth 2.1],
+    The _Global client profiles_ are client profiles pre-configured in {project_name} by default. They are pre-configured to be compliant with standard security profiles like *FAPI* and *OAuth 2.1*  in the link:https://www.keycloak.org/guides#securing-apps[securing apps] section,
```

Comment:
Same like my other comment (can be hopefully attribute instead of hardcoded link).